backend/integrations/discord/cogs.py

```
@@ -391,3 +399,124 @@ async def list_indexed_repos(self, interaction: discord.Interaction):
+                        await user.send(embed=build_final_handoff_embed())
+                    except Exception:
+                        pass
+                    return "auth_unavailable"
+
+            # Send welcome DM with actions (auth_url may be None when button disabled)
+            embed = self._build_welcome_embed(user)
+            await user.send(embed=embed, view=OnboardingView(auth_url))
+            return "onboarding_sent"
+        except discord.Forbidden:
+            return "dm_forbidden"
+        except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)